test/runtests.jl

```diff
@@ -19,28 +19,36 @@ end
     
         sodium_test_particle = MonteCarloCollisions.Particles("sodium_test_particle", 23*amu, q_e, 2)
     
-        MonteCarloCollisions.init_thermal(sodium_test_particle, 10)
+        MonteCarloCollisions.init_thermal(sodium_test_particle, 9)
     
         MonteCarloCollisions.init_thermal(fluor_test_particle, 10)
     
-        MonteCarloCollisions.init_thermal(oxygen_test_particle, 10)
+        MonteCarloCollisions.init_thermal(oxygen_test_particle, 11)
     
         scatter_v = MonteCarloCollisions.scatter(sodium_test_particle.list[1].v, fluor_test_particle.list[1].v, sodium_test_particle.m, fluor_test_particle.m, norm(sodium_test_particle.list[1].v - fluor_test_particle.list[1].v), 0)
             
         @test fluor_test_particle.list[1].v != scatter_v
 
-        @test energy(sodium_test_particle) > energy(fluor_test_particle) > energy(oxygen_test_particle)
+        @test MonteCarloCollisions.energy(sodium_test_particle) > MonteCarloCollisions.energy(fluor_test_particle) > MonteCarloCollisions.energy(oxygen_test_particle)
```

Comment:
Neni mi jasny vyznam tohoto testu: sodium/fluor/oxygen_test_particle atd... jsou pary nahodne vygenerovanych termalnich castic, takze pro jejich stredni energie sice nejaka (jina) nerovnost plati, ale pro jednotlive castice to klidne muze byt nahodou obracene. Pokud byste vygeneroval mnohem vice castic nez 2, tak by mohlo fungovat:
```suggestion
        @test sum(MonteCarloCollisions.energy(oxygen_test_particle)) > sum(MonteCarloCollisions.energy(fluor_test_particle)) > sum(MonteCarloCollisions.energy(sodium_test_particle))
```